### PR TITLE
Update README with SDK Beta Vcpkg Registry information

### LINKED ISSUE #2852: Set up vcpkg private registry
IMPORTANT: include public releases from vcpkg there as well!

### LINKED ISSUE #2853: Update readme for customers on how to consume betas from private vcpkg registry
Pre-requisite: #2852

### PR DESCRIPTION
**Pull request by antkmsft**
Closes #2852.
Closes #2853.

### CODE REVIEW

**Review by gearama — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -57,6 +57,12 @@ target_link_libraries(<your project name> PRIVATE Azure::azure-storage-blobs)
 
 You can reference this [vcpkg Quick Start](https://github.com/microsoft/vcpkg#quick-start-windows) for more details.
 
+#### Getting Beta Releases in Vcpkg
+
+Official vcpkg registry may have beta versions of Azure SDK client libraries, up until a given library gets released as General Availability (GA) version. After that, we don't publish post-GA beta releases of that library in the official registry.
```
Comment (ahsonkhan) [inline README.md:62]: We should use "stable"/"beta" instead of GA/preview.

Please fix the wording to use stable here.

**Pull request merged**